packages/bindings/src/mocks/mock_apps.rs

```diff
@@ -1,262 +0,0 @@
-//! Contains integration test utils for desmos custom modules.
```

Comment:
The current cw-multi-test doesn't provide stargate feature, in addition, I think the current integration test is not recommended since cw-multi-test can not represent real situation with the chain.

To have a integration test, we should recommend developer to run a new chain.

Reply:
Can you elaborate more why you think that `cw-multi-test` can not represent real situations with the chain?
Also how hard do you think is to update `cw-multi-test` to support the stargate feature?

Reply:
Since what cw-multi-test actually do is mock a application with states in rust, it is not a real chain. For example, it has own Bank module and store every bank related stuff inside the mock module, see:
https://github.com/CosmWasm/cw-multi-test/blob/main/src/bank.rs#L33

Therefore, from my side, if a contract need a integration test for real multi contract interaction, the best way is still running a chain then having a script to interact with chain, like we have `binding-test` to test all the generated msgs and queries.

To make cw-multi-test support stargate feature, we need to build a custom module to handle stargate msg and queries, and also handle the response logic. I think it can be done in two weeks, just don't see any benefit to use cw-multi-test.